src/recommendationservice/requirements.in

```diff
@@ -8,4 +8,5 @@ opencensus-ext-grpc==0.7.1
 python-json-logger==0.1.11
 google-cloud-profiler==1.1.2
 requests==2.27.1
-urllib3==1.26.5
\ No newline at end of file
+urllib3==1.26.5
+pyyaml==6.0
```

Comment:
**Nitpick:**
We could leave a comment here similar to
```
pyyaml==6.0 # This is not a direct dependency. We will need to keep an eye on google-python-cloud-debugger's use of pyyaml, update google-python-cloud-debugger, and eventually remove this line.
```
or maybe create a GitHub issue.
I just want to make sure the work that's yet to be done is visible to our team. :)

Reply:
I've created a new issue to track this: https://github.com/GoogleCloudPlatform/microservices-demo/issues/713
So we can do ahead and merge this pull-request. :)
Thanks again, Don, for this PR and the thanks, Olivier, for reviewing!